tests/mock_connector.py

```diff
@@ -22,3 +22,10 @@ def keys(self, db_id, pattern):
 
     def get_all(self, db_id, key):
         return MockConnector.data[key]
+
+    def delete_all_by_pattern(self, db_id, pattern):
```

Comment:
>delete_all_by_pattern [](http://example.com/codeflow?start=8&length=21)

This mock function only work on special use case, not a general implementation. Please consider copy the logic from codebase, and maybe implement a mock delete() function.